### PR TITLE
Adapter deprecation update blog

### PR DESCRIPTION
**Pull request by stianst**

### CODE REVIEW

**Review by jonkoops — APPROVED** [review]
LGTM

File: blog/2023/adapter-deprecation-update.adoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+* [Spring Security]https://docs.spring.io/spring-security/reference/servlet/oauth2/index.html - OAuth and OpenID Connect support in Spring
+* [Pac4j]https://www.pac4j.org/ - The Java security framework to protect all your web applications and web services
+
+Neither of these have support for Keycloak Authorization Services though, which is why we are planning to introduce a
+generic Java client libraries for Authorization Services that can be leveraged with other OpenID Connect client libraries.
+Expect this to be delivered in Keycloak 22.
+
+The Keycloak Java adapters will remain for a while though, at least towards the end of the year, but likely not be removed
+until early 2024. At the same time don't expect the adapters to be updated to be updated to support newer versions of
+Tomcat, Jetty, WildFly, or Spring.
```
Comment (mposolda) [inline blog/2023/adapter-deprecation-update.adoc:30]: We can also add some sentence like: `Also don't expect adapters to be updated with new OIDC/OAuth capabilities added to Keycloak server.` ?

But leaving to you if you prefer to add something like this or not.

**Review by mposolda — COMMENTED** [review]
@stianst LGTM. Just added 2 inline suggestions, which I think are typos. Also added one inline comment.

**Review by jonkoops — APPROVED** [review]

File: blog/2023/adapter-deprecation-update.adoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+* [Quarkus OIDC]https://quarkus.io/guides/security-openid-connect - OpenID Connect support for Quarkus applications
+* [Spring Security]https://docs.spring.io/spring-security/reference/servlet/oauth2/index.html - OAuth and OpenID Connect support in Spring
+* [Pac4j]https://www.pac4j.org/ - The Java security framework to protect all your web applications and web services
+
+Neither of these have support for Keycloak Authorization Services though, which is why we are planning to introduce a
+generic Java client libraries for Authorization Services that can be leveraged with other OpenID Connect client libraries.
+Expect this to be delivered in Keycloak 22.
+
+The Keycloak Java adapters will remain for a while though, at least towards the end of the year, but likely not be removed
+until early 2024. At the same time don't expect the adapters to be updated to support newer versions of
```
Comment (stianst) [inline blog/2023/adapter-deprecation-update.adoc:29]:
```suggestion
until early 2024. At the same time don't expect the adapters to be updated in terms of adding new features, enhancements, or supporting newer versions of
```

**Review by jonkoops — APPROVED** [review]

**Review by mposolda — APPROVED** [review]

File: blog/2023/adapter-deprecation-update.adoc [hunk 3]
```diff
@@ -0,0 +1,48 @@
+:title: Update on deprecation of Keycloak adapters
+:date: 2023-03-29
+:publish: true
+:author: Stian Thorgersen
+
+In 2022 we announced the deprecation of https://www.keycloak.org/2022/02/adapter-deprecation.html[deprecating Keycloak adapters], with
+a plan to stop delivering most adapters in https://www.keycloak.org/2022/03/releases.html[Keycloak 19].
+
+As we have not been able to make sufficient progress on finding alternatives and work on supporting material to help
+migrating away from Keycloak adapters we are extending the life of the Keycloak adapters.
```
Comment (stianst) [inline blog/2023/adapter-deprecation-update.adoc:10]:
```suggestion
migrating away from Keycloak adapters we are extending the life of the Keycloak adapters.

The plan is still to eventually stop delivering bespoke Keycloak adapters in the future, but we will do this in a more gradual process than previous laid out. 

We still strongly belive that the community as a whole are better served by us focusing more on the Keycloak server with better compliance and support for specifications such as OAuth 2.0 and OpenID Connect, while leaving the integration in various programming languages and frameworks to the relevant communities, which will be able to deliver more feature rich and better integrated libraries than we will ever be able to.
```

Comment (mposolda) [reply blog/2023/adapter-deprecation-update.adoc:10]: +1 to this addition

**Pull request merged**